atd-vze/src/queries/notes.js

```diff
@@ -13,3 +13,20 @@ export const GET_NOTES = gql`
     }
   }
 `;
+
+export const INSERT_NOTE = gql`
+  mutation InsertNote($note: String!, $crashId: Int!, $userEmail: String){
+    insert_notes(objects: {
+      text: $note
+      crash_id: $crashId
+      user_email: $userEmail
+    }) {
+      returning {
+        crash_id
+        text
+        date
+        user_email
+      }
+    }
+  }
+`;
```

Comment:
It's super not a big deal, but if you want, can you put a line feed at the end of the file? It's a hold-over convention from the 80s, but it persists as part of the POSIX standard.

Reply:
SO link on Frank's suggestion here:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
interesting, i absolutely will! thanks for this info :)